src/kata7.getEmployerRole.js

```diff
@@ -1,3 +1,9 @@
-const getEmployerRole = (employeeName, employees) => {};
+const getEmployerRole = (employeeName, employees) => {
+  for (let i = 0; i < employees.length; i++) {
+    if (employees[i].name === employeeName) {
+      return employees[i].role;
+    }
+  }
```

Comment:
Rather than a for loop, we can use the built in find method to discover any matches:
```suggestion
  const search = employees.find(employee => employee.name === employeeName);
  return search === undefined ? "Not an employee" : search.role;
```

This would also account for the case in which the employee is not found, which the current code doesnt take into account